<launch>
  <arg name="x" default="-2.0"/>
  <arg name="y" default="0.0"/>
  <arg name="theta" default="0.0"/>
  <arg name="step_length" default="0.5"/>
  <arg name="step_size" default="30"/>

      <node pkg="move_turtlebot" type="move_turtlebot_node" name="random_move" output="screen">
        <param name="x" value ="$(arg x)" />
        <param name="y" value ="$(arg y)" />
        <param name="theta" value ="$(arg theta)" />
        <param name="step_length" value ="$(arg step_length)" />
        <param name="step_size" value ="$(arg step_size)" />
      </node>

</launch>
